src/components/diff-blocks.component.js

```diff
@@ -12,42 +13,47 @@ type Props = {
   onPress: Function,
 };
 
-const styles = StyleSheet.create({
-  container: {
-    flexDirection: 'row',
-    alignItems: 'center',
-  },
-  linesChanged: {
-    flexDirection: 'row',
-    alignItems: 'center',
-  },
-  numAdditions: {
-    marginRight: 3,
-    ...fonts.fontPrimarySemiBold,
-    color: colors.green,
-    letterSpacing: 1,
-  },
-  numDeletions: {
-    marginRight: 2,
-    ...fonts.fontPrimarySemiBold,
-    color: colors.red,
-    letterSpacing: 1,
-  },
-  block: {
-    width: 7,
-    height: 7,
-    marginLeft: 1,
-  },
-  greenBlock: {
-    backgroundColor: colors.green,
-  },
-  redBlock: {
-    backgroundColor: colors.darkRed,
-  },
-  greyBlock: {
-    backgroundColor: colors.greyMid,
-  },
-});
+const Container = styled.Component`
+  flex-direction: 'row';
+  align-items: 'center';
+`;
```

Comment:
@AstroBoogie Can we leverage [`withComponent`](https://www.styled-components.com/docs/api#withcomponent)? For example, define another component and keep the condition judgement in line 80.
```
const TapableContainer = Container.withComponent(TouchableOpacity);
```